src/index.js

```diff
@@ -808,7 +824,7 @@ const validator = (schema, opts = {}) => compile(schema, schema, opts, Object.cr
 
 const parser = function(schema, opts = {}) {
   // strong mode is default in parser
-  const validate = validator(schema, { mode: 'strong', ...opts })
+  const validate = validator(schema, { mode: 'strong', ...opts, jsonCheck: false, isJSON: true })
```

Comment:
Leftover from debugging?

Reply:
No, parser always knows it got json and doesn't need an extra check. Because the data already came from json.parse.

Perhaps throwing if those are present in options would be a good idea, I'll add that.